### ISSUE TITLE
Cannot run linux version in Alpine

### ISSUE THREAD

**Issue by ChristophShyper** [minamijoyo/hcledit#10]
Hi. Have you tested your binary under dockerized Alpine?

I'm using following installation and it doesn't work.
```
(...)
	&& curl -L "$( curl -Ls https://api.github.com/repos/minamijoyo/hcledit/releases/latest | grep -o -E "https://.+?_linux_amd64.tar.gz" )" \
    -o hcledit.tar.gz \
	&& tar -xf hcledit.tar.gz hcledit \
	&& mv hcledit /usr/bin/hcledit \
  && chmod +x /usr/bin/hcledit \
  && chown $(id -u):$(id -g) /usr/bin/hcledit \
(...)
```

I'm getting `bash: ./hcledit: No such file or directory` error.

Can you please advise?

**Comment by minamijoyo** [comment]
@ChristophShyper Thank you for reporting the issue!

My guess is that some compile options are missing for static link, which is a common mistake in alpine.
I'll check it later. Thanks!

**Comment by minamijoyo** [comment]
@ChristophShyper Fixed in v0.1.1

**Comment by ChristophShyper** [comment]
Great! Thank you very much for such quick fix. Confirming it works properly now.
